frontend/templates/product/sections/ProductReviewsSection/index.tsx

```diff
@@ -122,7 +122,6 @@ export function ProductReviewsSection({
                            <ProductReviewLineItem
                               key={review.reviewid}
                               review={review}
-                              variantSku={selectedVariant.sku}
```

Comment:
This prop is still used, though I'm not sure exactly *why* it was in the conditional, we should probably just drop it.

Reply:
f6e90815cb01adbb52c0ec655209951c9e0e5e2d. 
Thanks Danny! Previously, we were using the `variantSKu = selectedVariant.sku` in the url so if the variantSku somehow ended up as undefined or null the link would break which is why we had it in the conditional. 
https://github.com/iFixit/react-commerce/pull/1573#discussion_r1165722871